WPMedia/fusion-news-theme-blocks
# Updated side-drawer behavior for rtl

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Updated the side drawer behavior so that it slides in from the right with RTL, and maintains the same behavior for LTR

## Jira Ticket

- [THEMES-1363](https://arcpublishing.atlassian.net/browse/THEMES-1363)

## Acceptance Criteria

When clicking on the sandwich menu on an RTL page, the sidebar needs to open up from the right side of the screen.  

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout 1363`
2. Make sure your components repo is up to date
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block`
4. Verify that the side drawer slides in from the left
5. Set "textDirection": "rtl" in the "siteProperties" object of the blocks.json (or on the HTML element in the dev tools).
6. Restart PageBuilder and verify that the side drawer slides in from the right

## Effect Of Changes

### Before

The drawer would always slide in from the left

### After

The drawer slides in from the left for ltr pages, and slides in from the right for rtl pages

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1363]: https://arcpublishing.atlassian.net/browse/THEMES-1363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
